app/controllers/passwords_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class PasswordsController < ApplicationController
+  def create
+    authorize :password
+
+    @password = Password.new(**create_params)
+
+    if @password.save
+      redirect_to '/'
```

Comment:
Missing a message saying that the password has been changed.

Reply:
Why redirect?
The user could stay in the same page, and after getting the 200, the message appears below the "save" button, or smth like that.

Reply:
(I think we should redirect to the login page, not `/`)

I don't think we should leave the user in the same page because the token will be marked as "used", and the form will no longer work. If you want to change it again, you'd need to request a new token (or to go to your profile and change it there, after logging in, when that's implemented)